assets/js/atomic/blocks/product-elements/tag-list/supports.js

```diff
@@ -10,8 +10,9 @@ export const supports = {
 			background: false,
 			link: true,
 		},
+		typography: {
+			fontSize: true,
+		},
+		__experimentalSelector: '.wc-block-components-product-tag-list',
 	} ),
-	typography: {
-		fontSize: true,
-	},
```

Comment:
same question here regarding existing users if we want to hide it behind the flag

Reply:
Same here